Wikipedia/Code/NSUserDefaults+WMFExtensions.swift

```diff
@@ -116,10 +116,17 @@ let WMFSendUsageReports = "WMFSendUsageReports"
     
     @objc var themeAnalyticsName: String {
         let name = string(forKey: WMFAppThemeName)
+        let darkMode =  bool(forKey: "SystemDarkMode")
```

Comment:
Can you make a property with this string, just so we're not hardcoding it twice? Maybe a private computed one just for this extension to see.